src/views/ForumView/index.tsx

```diff
@@ -213,7 +214,8 @@ export const ForumView = (props: ForumViewProps) => {
         disabled={!permission.readAndWrite}
         onClick={() => {
           setShowNewForumModal(true);
-        }}>
+        }}
+      >
```

Comment:
minor comment: tip for close to launch type fixes - we should try to save whitespace / syntax changes to a min so that the PR is less expansive for reviewers..

Reply:
agreed, this is coming from auto-linter. better to have a consistent linter config or a linter pre-commit hook.